### ISSUE TITLE
Error messages are difficult to read.

### ISSUE THREAD

**Issue by ckeeney** [isaachinman/next-i18next#256]
## Occurs in next-i18next version 0.37.0
```
# yarn.lock
next-i18next@^0.37.0:
  version "0.37.0"
  resolved "https://registry.yarnpkg.com/next-i18next/-/next-i18next-0.37.0.tgz#d82d69ae69cc4617be4385168a348795646bb59a"
  integrity sha512-QGyTJy96gxSuU6bpTSag6d5VAbUVIPttPgWFWu0Exk22PZ/iabIV+XcaB9h8RFBtJMn8bu+VD3jdhQ3mIZkGlA==
  dependencies:
    detect-node "^2.0.4"
    hoist-non-react-statics "^3.2.0"
    i18next "^15.0.0"
    i18next-browser-languagedetector "^3.0.0"
    i18next-express-middleware "^1.5.0"
    i18next-node-fs-backend "^2.1.0"
    i18next-xhr-backend "^2.0.0"
    path-match "^1.2.4"
    pretty-error "^2.1.1"
    prop-types "^15.6.2"
    react-i18next "
```

## Steps to reproduce
Do not include `namespacesRequired` to get a warning about the performance implications.

Here is the error rendered in my Webstorm terminal with the Darcula theme.
![image](https://user-images.githubusercontent.com/8799010/55666083-312f3d80-57fe-11e9-81fd-bc56f66284ba.png)

Sometimes my shell is quirky so I thought maybe this is my fault, but here is a screenshot of the same errors rendered in the Chrome developer console in the default theme:

![image](https://user-images.githubusercontent.com/8799010/55666119-c29eaf80-57fe-11e9-880d-6a17bc78b9f8.png)

 - Browser: `Chrome Version 73.0.3683.86 (Official Build) (64-bit)`

**Comment by isaachinman** [comment]
Sorry - what is the actual request here?

**Comment by ckeeney** [comment]
The error messages that come from this library are extremely difficult to read.  See the screenshots above.  This happens for both server side errors rendered on my terminal and client-side errors rendered in chrome developer tools.

I think we should provide some kind of theme for `pretty-error` or stop using it.  Right now I can only read those error messages by highlighting the error text.

**Comment by isaachinman** [comment]
Let's stop using it. I'll remove it sometime this week. Thanks.

**Comment by isaachinman** [comment]
Fixed by #259.
